src/callbacks/onResize/highlightMarks.js

```diff
@@ -0,0 +1,60 @@
+import { select } from 'd3';
+
```

Comment:
Like the new pseudo-stroke you've added to highlight lines. Thinking it might be good to be consistent with how we handle highlighting styles between lines and points. Currently we have: 
- Points
  - not highlighted = colored stroke, white fill
  - highlighted = black stroke,  colored fill
- lines
  - not highlighted = no stroke, colored fill
  - highlighted = black stroke (well, pseudo-stroke), colored fill.

I propose the following for both lines and points: 
- Not highlighted = colored fill , no stroke 
- highlighted = optional fill based on config.highlight_color (default:white), colored stroke

Reply:
I've created an issue to implement this. #87 - lets do it in a different branch.

Reply:
Handled in #83 .